bash_scripts/activate_idf_template.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+# Function to print environment variables
+print_env_variables() {
+    echo "IDF_PATH={{idf_path}}"
+    echo "IDF_TOOLS_PATH={{idf_tools_path}}"
+    echo "IDF_PYTHON_ENV_PATH={{idf_tools_path}}/python/"
+    echo "ESP_IDF_VERSION={{idf_version}}"
+}
```

Comment:
To align with the idf build system, it’s better to return all the environment variables provided by the idf_tools.py export command.

For example,  IDF_COMPONENT_MANAGER=1 is used by the idf build system to resolve and install dependencies mentioned in the idf_component.yml file